jekyll/_cci2/server-3-install.adoc

```diff
@@ -14,7 +14,16 @@ Before you begin with the CircleCI server v3.x installation process, ensure all
 
 toc::[]
 
-## Step 1 - Install KOTS
+## Phase 2 - Core Services Installation
+CircleCI server v3.x uses link:https://kots.io/[KOTS] from link:https://www.replicated.com/[Replicated] to manage and distribute server v3.x. KOTS is a kubectl plugin. To install the latest version, you can run 'curl https://kots.io/install | bash'
+
+Ensure you are running at least the minimum version (KOTS 1.47.3) by running `kubectl kots version`.  
```

Comment:
Elsewhere I think we say KOTS 1.44.1 minimum, shall I use 1.47.3 instead?

Reply:
@nathanfishcircleci

Reply:
yes. 1.47.3 is minimum. The most recent release is 1.50.3.